core/lib/dal/src/blocks_dal.rs

```diff
@@ -2394,6 +2396,29 @@ impl BlocksDal<'_, '_> {
         .flatten())
     }
 
+    pub async fn get_sealed_at(
```

Comment:
Bikeshedding: Despite the argument making it clear what this method is about, I'd still name it more specifically, like `get_batch_sealed_at()`; other methods in this module usually specify entities they concern.

Reply:
fixed